assets/src/application/ui/input/InlineEditInput/InlineEditInfinity.tsx

```diff
@@ -8,22 +8,22 @@ import { isInfinite } from '@application/services';
 import TabbableText from './TabbableText'
 import { TextProps } from './types';
 
-const Preview: React.FC<InlineEditPreviewProps> = ({ value, onRequestEdit, isEditing }) => {
-    const isInfinity = isInfinite(value);
-    const className = classNames('ee-inline-edit__infinity', {
-        'ee-infinity-sign': isInfinity,
-    });
+const Preview: React.FC<InlineEditPreviewProps> = ({ value, onRequestEdit, isEditing, ...props }) => {
```

Comment:
I bet this was edited in PhpStorm, for some reasons prettier from there is behaving a bit different than in VS Code

Reply:
nope that was VS Code... didn't notice that had been affected

Reply:
I noticed this because I also had this issue and upon saving my VS Code wasn't adding semicolon, even if in `.prettierrc` `semi` property is set to true and as a solution I've removed VS Code entirely with all is settings and installed again and it worked:

![Peek 2020-06-30 20-36](https://user-images.githubusercontent.com/1477453/86158514-8cd79c00-bb11-11ea-8746-5e3fb7b812fd.gif)